packages/components/src/styles/components/side-nav/header.scss

```diff
@@ -7,40 +7,7 @@
 // SIDE-NAV > HEADER
 //
 
-@use "../../mixins/focus-ring" as *;
-
-@mixin hds-side-nav-icon-button($add-visible-border: false) {
-  color: var(--token-side-nav-color-foreground-strong);
-  background-color: transparent;
-  border: 1px solid transparent;
-  border-radius: var(--token-side-nav-body-list-item-border-radius);
-  cursor: pointer;
-
-  @if ($add-visible-border) {
-    border-color: var(--token-color-palette-neutral-500);
-  }
-
-  &:focus,
-  &.mock-focus {
-    @include hds-focus-ring-with-pseudo-element($top: -1px, $right: -1px, $bottom: -1px, $left: -1px);
-  }
-
-  &:hover,
-  &.mock-hover {
-    color: var(--token-side-nav-color-foreground-strong); // to avoid overrides by specificity (eg. `a:hover`)
-    background: var(--token-side-nav-color-surface-interactive-hover);
-  }
-
-  &:active,
-  &.mock-active {
-    color: var(--token-side-nav-color-foreground-strong); // to avoid overrides by specificity (eg. `a:hover`)
-    background: var(--token-side-nav-color-surface-interactive-active);
-
-    @if ($add-visible-border) {
-      border-color: var(--token-color-palette-neutral-400);
-    }
-  }
-}
+@use "../../mixins/interactive-dark-theme" as *;
```

Comment:
By replacing the original mixin with this new shared one, the SideNav color tokens are no longer being used.

@didoo Please let me know what you think of this. (Is it a bad idea to use a shared mixin?)

Reply:
> the SideNav color tokens are no longer being used.

This is a tricky one! 
In theory we could deprecate and then remove them, and B) use the raw "palette" colors in the mixin like you're doing, or B) introduce a new set of tokens, more generic, and deprecate and then remove them. 
In reality these tokens are used in some our consumers codebases: https://github.com/search?q=%22--token-side-nav-color%22++user%3Ahashicorp+-repo%3Ahashicorp%2Fdesign-system&type=code&ref=advsearch
So the question becomes: what is the _correct™_ and most clean way of doing it (preserving the shared mixin, which I think is a good idea)? 🤔

Reply:
Thanks, I'll have to think more...

Reply:
Alright so what I ended up doing for now was that I updated the mixin to include an `$is-within-side-nav` option with conditionals that will use the sidenav tokens instead of the general tokens. Maybe a bit messy, but what do you think?

Reply:
I'll have a look tomorrow

Reply:
Update: I'm leaning towards keeping things simple for phase one and avoiding introducing new color tokens while designs are still in flux.